operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys.md

```diff
@@ -72,11 +73,20 @@ The Kubernetes image `k8s-image` is used by the master and worker nodes.
    ncn-m# unsquashfs -d k8s/${K8SVERSION}/filesystem.squashfs k8s/${K8SVERSION}/filesystem.squashfs.orig
    ```
 
+1. If the image being modififed contains the default SSH keys for the root user and/or the default
+   SSH host keys, remove them now.
```

Comment:
Might add context re: first time install vs. rotating.

Reply:
Yeah, I wasn't sure how to word this one due to the fact that customers are not likely doing a fresh install of 1.0.x these days. If they were doing a fresh install, and following the docs, the images in s3 will have already been scrubbed.  So "fresh install" is less relevant than "never modified / still contains default post-install".  That said, I'll think more a bit on rewording it.